src/config.ts

```diff
@@ -10,10 +11,11 @@ export const constants = {
   logStreamApiPath: "/api/logstream/application/functions/function/",
   masterKeyApiPath: "/api/functions/admin/masterkey",
   providerName: "azure",
+  rollbackEnabled: false,
```

Comment:
the README says the default is true -- whichever we choose, just need to be consistent.

Reply:
Right, the default in this PR is false just to not break everything. The next PR will set this to true